src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/ValidationRules.java

```diff
@@ -93,7 +93,7 @@
             "Arrival predictions for each trip in the feed must match the sequential order for the trips in the block. For example, if we have trip_ids 1, 2, and 3 that all belong to the same block, and the vehicle trips trip 1, then trip 2, and then trip 3, the arrival predictions should increase chronologically for trips 1, 2, and 3. For example, trip 3 predictions shouldn't be earlier in the feed than trip 2 predictions.",
             "predictions are not ordered by appearance in block");
 
-    public static final ValidationRule E015 = new ValidationRule("E014", "ERROR", "All stop_ids referenced in GTFS-rt feeds must have the location_type = 0",
+    public static final ValidationRule E015 = new ValidationRule("E015", "ERROR", "All stop_ids referenced in GTFS-rt feeds must have the location_type = 0",
```

Comment:
Thanks for catching this!  I missed changing this in https://github.com/CUTR-at-USF/gtfs-realtime-validator/commit/4444259605742d9527d3e89ff9c454bafd6c45e9 for #146.